### PR TITLE
Login Form

### PR DESCRIPTION
**Pull request by catarinalouza**

### CODE REVIEW

**Comment by catarinalouza** [comment]
Thought I had forgotten something so closed the pull request, but everything was fine so reopened it.

File: client/public/index.html [hunk 1]
```diff
@@ -23,26 +23,39 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+<<<<<<< HEAD
+  <title>Ingredient Tracker</title>
```
Comment (saritaSchen) [inline client/public/index.html:27]: are there two page titles now?

**Review by saritaSchen — COMMENTED** [review]
hey, I have one comment in index.html

File: client/public/index.html (hunk 1 above)
Comment (catarinalouza) [reply client/public/index.html:27]: apparently yes but not sure why, didn't even open the index.html for this 😆

Comment (saritaSchen) [reply client/public/index.html:27]: maybe after "git merge main" you said to keep both?

Comment (catarinalouza) [reply client/public/index.html:27]: yes most likely.. I always do the accept incoming changes or whatever so it probably added that second one.

Comment (saritaSchen) [reply client/public/index.html:27]: alright, it is definitely good to always check :D

**Review by saritaSchen — APPROVED** [review]
I deleted the doubled <title> tag myself and hope it didn't cause any trouble

**Pull request merged**